config/initializers/sentry.rb

```diff
@@ -0,0 +1,13 @@
+Sentry.init do |config|
+  config.dsn = ENV['sentry_key']
```

Comment:
Sentry by default takes dsn from `SENTRY_DSN` key and I would recommend to stay with the default.

Reply:
You're right, thank you.